### PR TITLE
Standardize Npgsql fork admonition text

### PR DESCRIPTION
**Pull request by mechanomi**

### CODE REVIEW

**Review by proddata — APPROVED** [review]
👍

File: README.rst [hunk 1]
```diff
@@ -4,9 +4,11 @@ CrateDB Npgsql Plugin
 
 A plugin for `Npgsql`_ that provides support for `CrateDB`_.
 
-Starting with CrateDB 4.2.0 the upstream `Npgsql`_ project can be used
-directly, and this forked version is no longer necessary.
+For CrateDB versions 4.2.0 and above, this plugin works with the standard
+upstream `Npgsql project`_.
```
Comment (mfussenegger) [inline README.rst:8]: Maybe the previous phrasing wasn't clear. But it wasn't about whether the plugin works with a forked version or the upstream project, but rather meant to say that starting with CrateDB 4.2.0 the plugin is not required at all.

Comment (mechanomi) [reply README.rst]: @mfussenegger aha! thanks. I pushed a fixup. can you check?

**Review by mfussenegger — APPROVED** [review]
:+1:

**Pull request merged**